contact-profile/contact-profile-project/contact-profile-app/components/ProfileImage.jsx

```diff
@@ -0,0 +1,50 @@
+import { Pencil } from './SVGs';
+
+const AVATAR_BG_COLOR = 'rgb(142 124 195)';
+
+const ProfileImage = ({ firstName = 'John Doe' }) => {
+  return (
+    <div
+      style={{
+        fontSize: '1rem',
+        color: 'white',
+        backgroundColor: AVATAR_BG_COLOR,
+        borderRadius: '50%',
+        height: '3rem',
+        width: '3rem',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      {firstName.slice(0, 1)}
```

Comment:
If the contact doesn't have a first name set, this throws an error. Maybe this should be empty if firstName is null or undefined?

I hit this when I was testing and created a form that only collected an email address, no name:

<img width="787" alt="image" src="https://github.com/HubSpot/cms-js-building-block-examples/assets/60455/407cfc7e-7f6e-403d-a355-e27c973d29df">